modules/preview-react/form-field/stories/examples/AllFields.tsx

```diff
@@ -60,14 +60,14 @@ export const AllFields = () => {
           <FormField.Hint>Error Message</FormField.Hint>
         </FormField.Container>
       </FormField>
-      <FormField as="fieldset">
-        <FormField.Label as="legend">Checkbox Legend</FormField.Label>
+      <FormField grow>
+        <FormField.Label>Checkbox Legend</FormField.Label>
```

Comment:
Should label still be a `legend` element? By default is `label`

Reply:
good catch